codegen/core/src/main/protobuf/akkaserverless/codegen/annotations.proto

```diff
@@ -0,0 +1,155 @@
+// Copyright 2021 Lightbend Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// All extension points for Akka Serverless
+
+syntax = "proto3";
+
+package akkaserverless;
+
+option java_package = "com.akkaserverless.codegen";
```

Comment:
This is now bringing all codegen annotation to the SDK project. 

I few things to note: 

* File to import is `akkaserverless/codege/annotations.proto`. Stays consistent with other annotations files.
* Proto package is still `akkaserverless`. this allow us to use `option(akkaserverless.codegen)`. We stay on the same namespace
* Java package must be `com.akkaserverless.codegen`, other we get two `com.akkaserverless.Annotations` files. Now we have  `com.akkaserverless.Annotations` (from framework) and `com.akkaserverless.codegen.Annotations` from the SDK
* All definitions are suffixed with `Def` to avoid conflicts with proto message in `akkaserverless`